src/App.js

```diff
@@ -1,16 +1,52 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import './App.css';
+
+const sender1 = chatMessages[0].sender;
+const sender2 = chatMessages[1].sender;
```

Comment:
Great idea to grab these constants! Just food for thought, what would this look like if the first two messages weren't from different senders? Or what would this look like for a conversation between more than one person?